### ISSUE TITLE
included configs allow yaml

### ISSUE THREAD

**Issue by yosefy** [bugy/script-server#721]
for now i see that i can't include yaml conf as in logs it try to parse it as json
is it possible to include yaml too?

thanks

**Comment by MNeill73** [comment]
This would be a bad idea.

Config files for WhateverProduct should accept a single syntax. JSON, or
YAML, or INI file, or whatever... But one format. A config file that allows
multiple syntaxes is just asking for config to break all over the place.

**Comment by yosefy** [comment]
yes but main script is yaml, but included can be only json
so it is exactly the state now [MNeill73](https://github.com/MNeill73)

**Comment by bugy** [comment]
Good point @yosefy, I guess we missed this scenario, when added support for yaml files.

**Comment by yosefy** [comment]
thanks [bugy](https://github.com/bugy)

also, can we just may be allow yaml in main config too?

not urgent at all but to be consistent

also it is pain, that there are no comments in json :)

**Comment by yosefy** [comment]
main is the runner  the main one that includes the rest
